vrr_client/src/Constants.ts

```diff
@@ -0,0 +1 @@
+export const VRR_TEST_API_BASE_URL = "https://openservice-test.vrr.de/";
```

Comment:
As discussed, we need to be able to override this URL during testing.

Reply:
You can provide any url in the constructor of the client.

Reply:
for E2E test we need to modify it using env vars. but I can implement this in my pr.